extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -190,7 +190,11 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
     this.element.setAttribute('data-amp-slot-index',
         this.win.ampAdSlotIdCounter++);
     this.lifecycleReporter_ = this.initLifecycleReporter();
-    this.ampAnalyticsConfig_ = null;
+    if (this.ampAnalyticsConfig_) {
+      this.ampAnalyticsConfig_.parentElement.removeChild(
```

Comment:
Yeah, I checked both AdSense and DoubleClick to make sure things rendered without throwing any errors. This should be fixed now.